client/src/visualiser-src/sorting-visualiser/animation-producer/SortsMergeAnimationProducer.ts

```diff
@@ -0,0 +1,67 @@
+import { mergeCodeSnippet } from "../util/codeSnippets";
+import { getX, getCx, getY } from '../util/helpers';
+import GraphicalSortsElement from "../data-structure/GraphicalSortsElement";
+import SortsAnimationProducer from "./SortsAnimationProducer";
+import { textCy, defaultColour, sortedColour, checkingColour } from "../util/constants";
+
+export default class SortsMergeAnimationProducer extends SortsAnimationProducer {
+    public renderMergeCode() {
+        this.renderCode(mergeCodeSnippet);
+    }
+
+    public highlightSorting(low: number, high: number, elemensList: GraphicalSortsElement[]) {
+        for (let i = low; i <= high; i += 1) {
+            this.addSequenceAnimation(elemensList[i].boxTarget.animate().attr({ fill: checkingColour }));
+            this.addSequenceAnimation(elemensList[i].numberTarget.animate().attr({ fill: checkingColour }));
+            this.addSequenceAnimation(elemensList[i].boxTarget.animate().attr({ stroke: checkingColour }));
+            this.addSequenceAnimation(elemensList[i].numberTarget.animate().attr({ stroke: checkingColour }));
+        }
+    }
+
+    public compareLowerandEqual(first: GraphicalSortsElement, second: GraphicalSortsElement): boolean {
+        // highlight color to green
+        this.addSequenceAnimation(first.boxTarget.animate(300).attr({ opacity: 0.6 }))
```

Comment:
Maybe change this to the blue colour in the constants file (currently in the master branch) rather than changing the opacity -- opacity changes aren't as obvious.

Reply:
Ok, I will change it